.github/workflows/ci.yaml

```
@@ -68,8 +68,70 @@ jobs:
         git diff --exit-code docs
       continue-on-error: true
 
+  unitwindows:
```

```suggestion
  unit-windows:
```
use dash (-) to separate words